pkg/storage/utils/volumes_test.go

```diff
@@ -120,4 +126,13 @@ var _ = Describe("GetVolumes", func() {
 			WithAllVolumes,
 		),
 	)
+
+	It("should trim backend volume name", func() {
```

Comment:
Are we testing the following three permutations?
1. length < 63. result name should not change
2. length = 63. result name should not change
3. length > 63. result name should change appropriately.
I think just checking the length is < 63 is not enough, we need to ensure the name doesn't change in option 1 and 2 as well.

Reply:
This test just checks the name that's arbitrarily created for the backend storage volume, I'm not worrying about keeping this name unchanged if it's smaller than 63 chars since it's not really used to create any resource. This was just an additional change to the PR for consistency, but it's unrelated to the fix.